org-formation/050-costs/_tasks.yaml

```diff
@@ -61,3 +61,6 @@ CostCategories:
   DefaultOrganizationBinding:
     Account: !Ref MasterAccount
     Region: !Ref primaryRegion
+  Parameters:
+    OwnerEmailRulesJSON: !Cmd 'wget -qO- https://cost-rules.finops.sageit.org/owner-emails'
```

Comment:
how about using the [sceptre-request-resolver](https://pypi.org/project/sceptre-request-resolver) instead?

also i get `name 'boto3' is not defined` when attempting to access that url?

Reply:
I've resolved the missing import for boto3, and plan on working on the !Request function as follow-up work